rel/app/macos/Sources/Livebook/Livebook.swift

```diff
@@ -92,6 +92,17 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         ElixirKit.API.stop()
     }
 
+    func applicationDidBecomeActive(_ aNotification: Notification) {
+        if (self.initialURLs == []) {
+            ElixirKit.API.publish("open", "")
+        }
+        else {
+            for url in self.initialURLs {
+                ElixirKit.API.publish("open", url.absoluteString)
+            }
+        }
```

Comment:
If we start Livebook by, say, opening up `foo.livemd` then that file URL is stored in initialURLs. Which means that if we later Spotligh Livebook, we'd open that file URL again. Which sometimes might be what we wanted, but I think it'd be predictable to always start at the Livebook home screen:

```suggestion
        ElixirKit.API.publish("open", "")
```